README.md

```diff
@@ -1,24 +1,93 @@
-# README
+<!-- @format -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+# Budget App
 
-* Ruby version
+> "Blog App" is a mobile web application where user can manage their budget from severals categories. In each categorie you can find a list of transactions with their corresponding amount in dollars($).
 
-* System dependencies
 
-* Configuration
+## Built With
 
-* Database creation
+- Ruby
+- Rails
+- Postgres
+- Rspec
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+## Live Demo
 
-* Services (job queues, cache servers, search engines, etc.)
+[Heroku](https://budget-capstone-1.herokuapp.com/)
 
-* Deployment instructions
+### Project Set up:
 
-* ...
+#### Step 1:
+
+- Open your command line and clone the repo locally
+
+```bash
+> git clone https://github.com/Nicolaswg/budget-app.git
+```
+
+- Then cd into the directory using
+
+```bash
+> cd budget-app
+```
+#### Step 2:
+
+- In the project directory, you should run the following command for installing all the required gems:
+
+```bash
+> bundle install
+```
+
+- then setup the db using the following command in the CL
+
+```bash
+> bin/rails db:setup
+```
+
+- Finally, you can use this command to start the live server and checkout the app!
+
+```bash
+> bin/rails s
+```
+
+## Testing:
+
+To perform tests, run the following command
+
+```bash
+> bundle exec rspec spec
+```
+
+## Authors
+
+👤 **Nicolas Gonzalez**
+
+- GitHub: [@Nicolaswg](https://github.com/Nicolaswg)
+- Twitter: [@Nicolas54146830](https://twitter.com/Nicolas54146830)
+- LinkedIn: [nicolas-gonzalez](https://www.linkedin.com/in/nicolas-gonzalez-8623461a0/)
+
+## Acknowledgments
+> Thanks to [Gregoire Vella](https://www.behance.net/gregoirevella) for providing the design of this project on [Behance](https://www.behance.net/)
+
+-[Original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome and highly encouraged!
+We feel that your input is what helps us grow so you're always Welcome :)
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+I would love to hear your thoughts and ideas 🖤
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] This MIT link is broken. It would be better to fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.